### PR TITLE
docs: translate ko/spec/ics-003-connection-semantics/README.md

### PR DESCRIPTION
**Pull request by junbeomlee**

### CODE REVIEW

**Comment by junbeomlee** [comment]
@mossid

File: translation/ko/spec/ics-003-connection-semantics/README.md [hunk 1]
```diff
@@ -0,0 +1,395 @@
...
+  counterpartyVersion: string): boolean {
+    // defined by host state machine
+}
+```
+
+이 명세의 다음 버전 또한 이 함수를 정의할 것입니다.
+
+### Sub-protocols
+
+이 ICS는 개방 핸드쉐이크 서브 프로토콜을 정의합니다. 연결이 열릴 때, 연결은 닫힐 수 없고 식별자는 재할당 될 수 없습니다 (이것은 패킷의 재발생 또는 인증 혼란을 방지해줍니다).
```
Comment (mossid) [inline translation/ko/spec/ics-003-connection-semantics/README.md]:
```suggestion
이 ICS는 개방 핸드쉐이크 서브 프로토콜을 정의합니다. 연결이 한번 열리고 나면, 연결은 닫힐 수 없고 식별자는 재할당 될 수 없습니다 (이것은 패킷의 재발생 또는 인증 혼란을 방지해줍니다).
```

File: translation/ko/spec/ics-003-connection-semantics/README.md [hunk 2]
```diff
@@ -0,0 +1,395 @@
...
+
+헤더 추적과 오동작 감지는 [ICS 2](../ics-002-client-semantics)에 정의되어 있습니다.
+
+![State Machine Diagram](../../../spec/ics-003-connection-semantics/state.png)
+
+#### 식별자 검증
+
+연결은 유일한 `Identifier` 접두어에 의해 저장됩니다. 검증 함수 `validateConnectionIdentifier`은 제공될 것입니다.
```
Comment (mossid) [inline translation/ko/spec/ics-003-connection-semantics/README.md]:
```suggestion
연결은 유일한 `Identifier` 접두어에 의해 저장됩니다. 검증 함수 `validateConnectionIdentifier`은 제공될 수도 있습니다.
```

File: translation/ko/spec/ics-003-connection-semantics/README.md [hunk 3]
```diff
@@ -0,0 +1,395 @@
...
+
+구현은 반드시 상대방이 제안한 버전 리스트에서 버전을 선택하는 `pickVersion` 함수를 정의해야 합니다.
+
+```typescript
+type pickVersion = ([]string) => string
+```
+
+#### Opening Handshake
+
+핸드쉐이크 개방 서브 프로토콜은 두 체인의 합의 상태를 초기화합니다.
```
Comment (mossid) [inline translation/ko/spec/ics-003-connection-semantics/README.md]:
```suggestion
핸드쉐이크 개방 서브 프로토콜은 두 체인이 서로에 대해 가진 합의 상태를 초기화합니다.
```

File: translation/ko/spec/ics-003-connection-semantics/README.md [hunk 4]
```diff
@@ -0,0 +1,395 @@
...
+Relayer | `ConnOpenTry` | B | (INIT, none) | (INIT, TRYOPEN)
+Relayer | `ConnOpenAck` | A | (INIT, TRYOPEN) | (OPEN, TRYOPEN)
+Relayer | `ConnOpenConfirm` | B | (OPEN, TRYOPEN) | (OPEN, OPEN)
+
+서브 프로토콜을 구현하는 두 체인의 개방 핸드쉐이크의 종료 시점에서, 다음과 같은 속성들이 유지됩니다:
+
+- 각 체인은 초기 행위자(actor)가 지정한 대로 서로의 올바른 합의 상태를 갖고 있습니다.
+- 각 체인은 다른 체인의 식별자를 알고 있으며, 이에 합의합니다.
+
+이 서브 프로토콜은 모듈로 안티 스팸 대책으로 허가 받을 필요는 없습니다.
```
Comment (mossid) [inline translation/ko/spec/ics-003-connection-semantics/README.md]:
```suggestion
이 서브 프로토콜은, 스팸 방지용 기준을 제외하고는, 허가식일 필요가 없습니다.
```

**Comment by cwgoes** [comment]
Bump @junbeomlee @mossid

**Comment by cwgoes** [comment]
Do you still intend to continue this? Otherwise I'll close this PR.

**Pull request closed without merging**